### ISSUE TITLE
Save files does not work. What can be DONE?

### ISSUE THREAD

**Issue by ysharoiko** [astrada/google-drive-ocamlfuse#181]
When i save file it does not appear on googl drive, 
although i can see it locally and get os.stat on it.
Waiting gives no result.
If i repeat action and save it again it goes ok.
What can be done to fix it?

**Comment by ysharoiko** [comment]
Made an experiment.

saved flie test-2.csv with system calls f.save(fname) from python. And check if it os.path.exists(fname)
Then I removed it from google interface.
Local system was still showing it as an existsing.
I repeated saving. It showed me no errors and os.path showed true.
And in google interface there were no file.
Than local file desappered - guess cache refreshed.
I did repeat my saving test with adding new files test-4.csv and test-5.csv
And these new files created exactly, but the file test-2.csv still did not created. But os.path on it showed true!!! 
However this file is not shown locally neither in google interface.
